### PR TITLE
Add changelog for 1.9.3

### PR DESCRIPTION
**Pull request by amacaskill**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation

> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Add changelog for 1.9.3. Also fixes errors in 1.8 and 1.7 changelog that the PR author is incorrect for some changes. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.9.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# v1.9.3 - Changelog since v1.9.2
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Fix missing libedit.so.2 error ([#1177](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1177), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
```
Comment (sunnylovestiramisu) [inline CHANGELOG/CHANGELOG-1.9.md:7]: There is also the Update go version to 1.20.3 for k/k 1.27: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1181

Comment (amacaskill) [reply CHANGELOG/CHANGELOG-1.9.md:7]: Added

**Comment by amacaskill** [comment]
@sunnylovestiramisu Can you please lgtm?

**Pull request merged**